Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -1753,6 +1886,27 @@ void ResourceLoadStatisticsStore::requestStorageAccess(SubFrameDomain&& subFrame
     });
 }
 
+void ResourceLoadStatisticsStore::setLoginStatus(RegistrableDomain&& domain, IsLoggedIn loggedInStatus, std::optional<LoginStatus>&& lastAuthentication)
+{
+    ASSERT(!RunLoop::isMain());
+
+    auto loginStatusToSet = lastAuthentication && lastAuthentication->hasExpired() ? std::nullopt : std::optional(WTFMove(lastAuthentication));
+    if (loginStatusToSet)
+        loginStatusToSet->setTimeToLive(WebCore::LoginStatus::TimeToLiveLong);
+
+    if (loggedInStatus == IsLoggedIn::LoggedIn)
+        insertLoginStatus(WTFMove(domain), loggedInStatus, WTFMove(lastAuthentication));
+    else if (loggedInStatus == IsLoggedIn::LoggedOut)
+        removeLoginStatus(WTFMove(domain));
+}
+
+bool ResourceLoadStatisticsStore::isLoggedIn(RegistrableDomain&& domain)
+{
+    ASSERT(!RunLoop::isMain());
+
+    return loginStatusExists(WTFMove(domain));
```

Comment:
`WTFMove()` is unnecessary, loginStatusExists() takes a `const RegistrableDomain&`